<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Token</class>
 <widget class="QDialog" name="Token">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Token</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Ingreso de token</string>
   </property>
   <widget class="QLabel" name="indicacion_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fila 3, columna 2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="token_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="validate_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
